src/README_OLD.md

```diff
@@ -0,0 +1,203 @@
+# EDITOR-SDK GETTING STARTED
+
+In this overview, we provide you with a small guide which should get you started on working / integrating with the SDK.
+After the getting started is working, you can look further at the documentation to search for methods and listers that you need.
+If you have any questions, do not hesitate to reach out to any of the chili team-members.
+
+Happy coding and enjoy the **spicy** ride.
+
+## General Structure
+
+The SDK consist of controllers which are individual Javascript classes and methods of these classes.
+Currently there are 7 different controllers:
+
+-   AnimationController
```

Comment:
same as above

Reply:
old readme, not used anymore but is there still as a reference, should be deleted soon